docs/content/preview/explore/change-data-capture/debezium-connector-postgresql.md

```diff
@@ -0,0 +1,1904 @@
+---
+title: Debezium connector for YugabyteDB (Logical Replication)
+headerTitle: Debezium connector for YugabyteDB (Logical Replication)
+linkTitle: Debezium connector for YBPG
+description: Debezium is an open source distributed platform used to capture the changes in a database.
+aliases:
+  - /preview/explore/change-data-capture/debezium-connector-yugabytedb-ysql-pg
+#   - /preview/explore/change-data-capture/debezium-connector
+#   - /preview/explore/change-data-capture/debezium
+  - /preview/explore/change-data-capture/debezium-connector-postgres
+  - /preview/explore/change-data-capture/debezium-YugabyteDB
+  - /preview/explore/change-data-capture/replication
+menu:
+  preview:
+    parent: explore-change-data-capture
+    identifier: debezium-connector-yugabytedb-pg
+    weight: 20
+type: docs
+rightNav:
+  hideH4: true
+---
+
+The Debezium YugabyteDB connector captures row-level changes in the schemas of a YugabyteDB database.
+
+The first time it connects to a YugabyteDB server or cluster, the connector takes a consistent snapshot of all schemas. After that snapshot is complete, the connector continuously captures row-level changes that insert, update, and delete database content and that were committed to a YugabyteDB database. The connector generates data change event records and streams them to Kafka topics. For each table, the default behavior is that the connector streams all generated events to a separate Kafka topic for that table. Applications and services consume data change event records from that topic.
+
+## Overview
+
+YugabyteDB’s [logical decoding](#reminder) feature was introduced in version 9.4. It is a mechanism that allows the extraction of the changes that were committed to the transaction log and the processing of these changes in a user-friendly manner with the help of an [output plug-in](#reminder). The output plug-in enables clients to consume the changes.
+
+The YugabyteDB connector contains two main parts that work together to read and process database changes:
+
+<!-- todo vaibhav: need sub-bullets -->
+* A logical decoding output plug-in. You might need to install the output plug-in that you choose to use. You must configure a replication slot that uses your chosen output plug-in before running the YugabyteDB server. The plug-in can be one of the following:
```

Comment:
Is there a use case where user might have to install the output plug-in on their own? My understanding is that we ship yboutput with the product and pgoutput is builtin, so customers don't have to install anything.

Reply:
No, the user doesn't need to install the plug-in on their own. I have reworded the statement now.